includes/wikia/api/tests/RecommedationTileApiControllerTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+class RecommendationTileApiControllerTest extends TestCase {
+	protected function setUp() {
+		parent::setUp();
+		require_once __DIR__ . '/../RecommendationTileApiController.class.php';
+	}
+
+	public function testRecommendationTileShouldReturnTheSameNumberOfValuesAsRequested() {
+		//given
+		$requestedIds = '1,8';
+		$objectUnderTest = new TestableRecommendationTileApiController();
+
+		$request = new WikiaRequest( [
+			WikisApiController::PARAMETER_WIKI_IDS => $requestedIds
+		] );
+		$response = new WikiaResponse( WikiaResponse::FORMAT_JSON, $request );
+
+		$objectUnderTest->setRequest( $request );
+		$objectUnderTest->setResponse( $response );
+
+		//when
+		$objectUnderTest->getDetails();
+
+		//then
+		$responseItems = $response->getVal( 'items' );
+		$responseItems = get_object_vars($responseItems);
+
+		$first = $responseItems['123_1'];
+		$this->assertEquals('title 1', $first['title']);
+		$this->assertEquals('test_wiki_name', $first['wikiName']);
+		$this->assertEquals('image_url_1', $first['thumbnail']);
```

Comment:
code should be spacey - see guidelines https://github.com/Wikia/guidelines/tree/master/PHP, there's an IntelliJ code style too :)